Gemfile

```diff
@@ -65,3 +66,7 @@ group :test do
   gem "capybara"
   gem "selenium-webdriver"
 end
+
+
+gem 'rspec-rails', '~> 6.1.0'
+gem 'faker'
```

Comment:
Do we need these gems in production?

Reply:
fixed it by moving them under test and development group

Reply:
moved rspec to test and ffaker to test and development